fixbackend/subscription/subscription_repository.py

```diff
@@ -97,7 +97,7 @@ def to_model(self) -> BillingEntry:
             id=self.id,
             workspace_id=self.workspace_id,
             subscription_id=self.subscription_id,
-            tier=SecurityTier(self.tier),
+            tier=ProductTier(self.tier),
```

Comment:
Don't we need a fallback for the previous values?

Reply:
Nobody was able to switch the tier - all users are in FreeAccount, which is still available.